client/a8c-for-agencies/sections/referrals/consolidated-view/index.tsx

```diff
@@ -106,6 +115,21 @@ export default function ConsolidatedViews( {
 						args: { month },
 						comment: 'month is the name of the current month',
 					} ) }
+					<InfoIconWithPopover>
+						<div className="consolidated-view__popover-content">
+							{ translate(
+								'When your client buys products or hosting from Automattic for Agencies, they are billed on the first of every month rather than immediately.' +
+									' We estimate the commission based on the active use for the current month. {{a}}Learn more about payouts and commissions{{/a}}.',
+								{
+									components: {
+										nbsp: <>&nbsp;</>,
```

Comment:
Hmm, where this is used?

Reply:
I totally missed this comment. Sorry, @andrii-lysenko. You are right, this is not required. I'll create a PR to fix this.